airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/ConnectionCell.module.scss

```diff
@@ -0,0 +1,8 @@
+.connectorItem {
+  width: calc(100% - 30px / 2);
+}
+
+.arrowItem {
+  flex-basis: 30px;
+  min-width: 30px;
```

Comment:
Is the idea to make this flex child exactly 30px? I would use `flex: 0 0 30px` here instead of setting width. Or is there a reason to set `min-width` explicitly?

Could we also add:

```
display: flex;
align-items: center;
justify-content: center;
```
So the arrow is nicely centered?